near/omni-bridge/src/lib.rs

```diff
@@ -892,6 +892,25 @@ impl Contract {
         }
     }
 
+    pub fn resolve_token_address(
+        &self,
+        token: &OmniAddress,
```

Comment:
```suggestion
        address: &OmniAddress,
```
The `token` and `token_id` are usually used for account id in our code.
For the OmniAddress, let's use `address`